content/applications/websites/website/configuration/google_search_console.rst

```diff
@@ -0,0 +1,30 @@
+=====================
+Google Search Console
+=====================
+
+Google Search Console is a free web service provided by Google that allows website owners to monitor,
+maintain, and troubleshoot their site's presence in Google Search results. It offers valuable insights
+into how Google views and interacts with your site, helping you optimize its performance.
+
+To enable GSC, go to `Google Search Console <https://search.google.com/search-console/welcome>`_,
+enter your domain or URL prefix, follow the instructions, and copy the code from Google.
+
+.. image:: google_search_console/add-domain-or-url-prefix.png
```

Comment:
Do we need to enter www. or is it not necessary in this case?

Reply:
not necessary